lib/person.rb

```diff
@@ -0,0 +1,34 @@
+require 'yaml'
+require './lib/library'
+$collection = YAML.load_file('./lib/data.yml')
```

Comment:
You should access the collection of books through an instance of the Library class. Not read it directly.